google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -416,14 +416,14 @@ void populateCondition(Rule.Condition condition) {
      * <a href="https://cloud.google.com/storage/docs/storage-classes">here</a>.
      */
     @Deprecated
-    public abstract Builder storageClass(String storageClass);
+    public abstract Builder storageClass(StorageClass storageClass);
```

Comment:
This would become isolated once my deprecated PR is merged. so I think it would be more efficient to merge the deprecated PR before this one.